BUTTON_IDEAS.md

```diff
@@ -61,5 +61,11 @@ When adding ideas, please add them in the right category (or add a new one if yo
 
 | status  | Button name                    | Description                          | Submitted by                                  |
 | ------- | ------------------------------ | ------------------------------------ | --------------------------------------------- |
+
```

Comment:
Remove this extra spacing as it ruins the table's format

Reply:
ok sure